src/DataTable/Table.jsx

```diff
@@ -1,16 +1,19 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import Spinner from 'react-bootstrap/Spinner';
```

Comment:
nit: Let's use the `Spinner` component exported from Paragon itself, instead of the straight pass through from `react-bootstrap`. The benefit is that we could more easily pass in a `screenReaderText` prop for a11y.

Reply:
Fixed, thanks. Adam, one question. Is it ok if `screenreadertext ` is used in lowercase, because if use uppercase, like `screenReaderText ` tests are failed?

Reply:
@vlasovmichael What about the tests fail with `screenReaderText` as the prop?

Passing `screenreadertext` in all lowercase would likely try to add `screenreadertext` as an attribute to the DOM node, which isn't right, or would otherwise be unhandled.

`screenReaderText` (camelCase) prop is a [valid prop for Paragon's `Spinner` component](https://github.com/edx/paragon/blob/master/src/Spinner/index.jsx#L8) so I'm unclear what about the tests would be failing here 🤔

Reply:
Fixed, thanks!